src/datalad_installer.py

```diff
@@ -823,7 +823,7 @@ def long_help(cls, progname: str, component: Optional[str] = None) -> str:
             width = max(map(len, cls.COMPONENTS.keys()))
             for name, cmpnt in sorted(cls.COMPONENTS.items()):
                 if cmpnt.OPTION_PARSER.help is not None:
-                    chelp = cmpnt.OPTION_PARSER.help
+                    chelp = cmpnt.OPTION_PARSER.help.splitlines()[0]
```

Comment:
What's the point of this change?

Reply:
so that in `datalad-installer --help` we have only the first line listed.  Without this I get

```
❯ src/datalad_installer.py --help
...
  miniconda                Install Miniconda VERSION is the version [...]
```